themes/custom/droopler_theme/scss/components/_d_p_counters.scss

```diff
@@ -108,6 +120,7 @@ $d-p-counters-background-opacity: 0.2 !default;
 
     .field--name-field-d-long-text {
       padding-top: 1.51rem;
+      padding-bottom: 1.51rem;
```

Comment:
Pytanie czy to nie powinien być padding nad polem z listą counterów? Czy takie rozwiązanie zadziała jeśli nie wypełni się długiego opisu?

Reply:
Bez tego long text który ma tylko p się przykleja
Dolny dodałem po wywaleniu margina dolnego po ostatnim p w long text